### DISCUSSION TITLE
Approved but not Verified

### DISCUSSION THREAD

**Discussion by kerolos-mohsen** [community/community#67246] [GitHub Education]
### Hi everyone,

guys i have singed for the github student pack . Congrats! Your request for [@kerolos-mohsen](https://github.com/kerolos-mohsen) was approved and your benefits will be available soon. it says that I am approved and will get the benefits soon but I got nothing , it has been more than 3 days so how much time does this verification take  ? + does this pack give me access to github copilot for free ?

**Comment by arafajs** [comment]
Hi @kerolos-mohsen 
According to the GitHub documentation, the activation process of Benefits for the GitHub Student Developer Pack usually takes up to 72 hours. However, it may take longer during peak times, such as during the start of a new semester.

**Comment by mantasu** [comment]
Hey, I had the same issue, you still may have to wait for some time since yeah there will be some delay before your benefits become active. When "Approved" tag becomes purple, you should be ready:

![Screenshot 2023-09-24 072637](https://github.com/community/community/assets/56790921/d80b71bb-8d34-424d-a682-39a0f986389c)

If you don't receive any benefits after 2 weeks, just open a [ticket](https://support.github.com/request/education?tags=docs-generic).
